fusor-ember-cli/app/routes/deployment.js

```diff
@@ -43,6 +44,62 @@ export default Ember.Route.extend(DeploymentRouteMixin, {
 
   },
 
+  loadOpenStack(controller, model) {
+    var self = this;
+    if (model.get('deploy_openstack') && !Ember.isBlank(model.get('openstack_undercloud_password'))) {
```

Comment:
@isratrade, this turned out to be kinda tricky and might be more straightforward the way it is.

In the use case that they haven't gotten to the OSP page and filled out the undercloud stuff the model.openstack_undercloud_password check prevents the load already.

In the use case that undercloud password is filled out, but someone has gone back to rhev/satellite and incorrectly filled it out, the OSP instantly responds.  However, if I check satellite/rhev, and something on that side has changed, we get into situations where I need to check osp but the data isn't loaded, forcing me to observe those values and trigger a load conditionally.

Is it ok if we leave it unless we find issues?